Dictionaries/eng_OCRFixReplaceList.xml

```diff
@@ -742,7 +735,6 @@
     <Word from="Iwanted" to="I wanted" />
     <Word from="Iwas" to="I was" />
     <Word from="Iwasjust" to="I was just" />
-    <Word from="Iwasjustu" to="I was just..." />

```

Comment:
I'll put it back in, then.

There's also `<Word from="readytolaunchu" to="ready to launch.." />`. Should I put that back too and just add the third dot?